vocabularies/party-relationship.ttl

```diff
@@ -0,0 +1,85 @@
+@prefix dc: <http://purl.org/dc/elements/1.1/> .
+@prefix dct: <http://purl.org/dc/terms/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix pyrl: <http://linked.data.gov.au/def/party-relationship/> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+

```

Comment:
Need to add in @prefix sdo: <https://schema.org/> .

Reply:
done.

Reply:
Does this line need the <> brackets for correct syntax?

@prefix sdo: <https://schema.org/> .

Reply:
Needs < > 
Github stripped these.

Reply:
Pesky. Changed.